demisto_sdk/commands/test_content/tests/test_context_test.py

```diff
@@ -126,8 +126,9 @@ def test_unmockable_playbook_passes_most_of_the_time(mocker, tmp_path):
 
     assert incident_test_mock.call_count == 3
     assert not build_context.tests_data_keeper.failed_playbooks
-    assert 'unmocked_playbook - Successful runs: 2/3' in build_context.tests_data_keeper.succeeded_playbooks
-    assert 'Test-Playbook was executed 3 times, and passed 2 times. Adding to succeeded playbooks.' == logs.info.call_args_list[14][0][0]
+    assert 'unmocked_playbook' in build_context.tests_data_keeper.succeeded_playbooks
+    assert any('Test-Playbook was executed 3 times, and passed 2 times. Adding to succeeded playbooks.' in [log_item][0][0] for
```

Comment:
isn't it supposed to be (same for all):
```suggestion
    assert any('Test-Playbook was executed 3 times, and passed 2 times. Adding to succeeded playbooks.' in log_item[0][0] for
```

if not, then you can do:
```suggestion
    assert any('Test-Playbook was executed 3 times, and passed 2 times. Adding to succeeded playbooks.' in log_item[0] for
```

Reply:
Done